package.json

```diff
@@ -70,5 +69,8 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "dependencies": {
+    "semver": "^6.3.0"
```

Comment:
Should this dependency be moved to devDependencies?

Reply:
yes!